[2022-04-14 09:12:59,290] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test_1_task_10s.task manual__2022-04-14T09:12:58.505069+00:00 [queued]>
[2022-04-14 09:12:59,302] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test_1_task_10s.task manual__2022-04-14T09:12:58.505069+00:00 [queued]>
[2022-04-14 09:12:59,302] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 09:12:59,303] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-14 09:12:59,303] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 09:12:59,317] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): task> on 2022-04-14 09:12:58.505069+00:00
[2022-04-14 09:12:59,323] {standard_task_runner.py:52} INFO - Started process 3234 to run task
[2022-04-14 09:12:59,325] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_1_task_10s', 'task', 'manual__2022-04-14T09:12:58.505069+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_one_task_10s.py', '--cfg-path', '/tmp/tmpz4ifb4i4', '--error-file', '/tmp/tmpm8vdzuxa']
[2022-04-14 09:12:59,325] {standard_task_runner.py:80} INFO - Job 24: Subtask task
[2022-04-14 09:12:59,370] {logging_mixin.py:109} INFO - Running <TaskInstance: test_1_task_10s.task manual__2022-04-14T09:12:58.505069+00:00 [running]> on host f66005c50d71
[2022-04-14 09:12:59,420] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=test_1_task_10s
AIRFLOW_CTX_TASK_ID=task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-14T09:12:58.505069+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-14T09:12:58.505069+00:00
[2022-04-14 09:13:09,424] {python.py:175} INFO - Done. Returned value was: None
[2022-04-14 09:13:09,434] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=test_1_task_10s, task_id=task, execution_date=20220414T091258, start_date=20220414T091259, end_date=20220414T091309
[2022-04-14 09:13:09,461] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-14 09:13:09,492] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
